invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page/creatibutors.less

```diff
@@ -2,12 +2,61 @@
  *   Copyright (C) 2020 CERN.
  *   Copyright (C) 2020 Northwestern University.
  *   Copyright (C) 2021 Graz University of Technology.
- *   Copyright (C) 2021 New York University.
+ *   Copyright (C) 2021-2022 New York University.
  *
  * Invenio RDM Records is free software; you can redistribute it and/or modify
  * it under the terms of the MIT License; see LICENSE file for more details.
  */
 
+.creatibutors {
+  margin: 0rem 0rem 0.5rem 0rem;
+  display: flex;
+  flex-wrap: wrap;
+  &:last-child {
+    margin-bottom: 0rem;
+  }
+
+  dt {
+    font-weight: bold;
+    margin: 0rem 0.25rem 0rem 0rem;
+
+    &.hidden {
+      display: none;
+    }
+  }
+
+  .creatibutor-wrap {
+    color: @mutedTextColor;
+    display: flex;
+    flex-wrap: wrap;
+    &:nth-child(n):not(:last-child) {
+      margin-right: 0.5rem;
+    }
+    .identifier-link, .group.icon {
+      margin: 0 1px 0 5px;
```

Comment:
We should use rem/em values to ensure the spacing stays consistent when scaling the screen.